zeekscript/cli.py

```diff
@@ -85,6 +85,10 @@ def do_write(source: bytes) -> None:
                     print_error(f"{fname}: {msg}")
                 else:
                     print_error(msg)
+
+                # Do not attempt to format code with errors.
+                do_write(script.source)
+                return 1
```

Comment:
This seems to wide, instead we should just not format format any nodes which have errors.